corehq/apps/domain/forms.py

```diff
@@ -2377,3 +2383,91 @@ def __init__(self, defaults=None, disable_input=False, **kwargs):
                     css_class="disabled"
                 )
             )
+
+
+class ManageAppReleasesForm(forms.Form):
+    app_id = forms.ChoiceField(label=ugettext_lazy("Application"), choices=(), required=False)
+    location_id = forms.ChoiceField(label=ugettext_lazy("Location"), choices=(), required=False)
+    version = forms.IntegerField(label=ugettext_lazy('Version'), required=False)
+
+    def __init__(self, request, domain, *args, **kwargs):
+        self.domain = domain
+        super(ManageAppReleasesForm, self).__init__(*args, **kwargs)
+        self.fields['app_id'].choices = self.app_id_choices()
+        if request.GET.get('app_id'):
+            self.fields['app_id'].initial = request.GET.get('app_id')
+        self.fields['location_id'].choices = self.location_id_choices()
+        if request.GET.get('location_id'):
+            self.fields['location_id'].initial = request.GET.get('location_id')
+        if request.GET.get('version'):
+            self.fields['version'].initial = request.GET.get('version')
+        self.helper = FormHelper()
```

Comment:
Use [HQFormHelper](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/crispy.py#L21) here to get standard HQ proportions. (It's new.)